README.md

```diff
@@ -48,6 +48,48 @@ gem 'rails'
 gem 'stripe'
 ```
 
+## Usage
+
+The library needs to be configured with your account's secret key which is
+available in the [Dashboard][dashboard]. Assign its value to `Stripe.api_key`
+and the library will send it along automatically with every request:
+
+``` ruby
+require "stripe"
+Stripe.api_key = "sk_test_..."
+
+# list charges
+Stripe::Charge.list()
+
+# retrieve single charge
+Stripe::Charge.retrieve(
+  "ch_18atAXCdGbJFKhCuBAa4532Z",
+)
+```
+
+You can also set a per-request key like in the example below. This is often
+useful for Connect applications that use multiple API keys during the lifetime
+of a process.
+
+Authentication is transparently handled for you in subsequent method calls on
+the returned object.
+
+For example:
+
+``` ruby
+require "stripe"
+
+Stripe::Charge.list(
+  {},
+  :api_key => "sk_test_..."
+)
```

Comment:
Cool. `stripe_account` seems like it would be useful to the same subset of people. I've amended the patch here to include that as well. Thanks!